src/app/views/query-runner/request/permissions/TabList.tsx

```diff
@@ -55,18 +56,27 @@ const TabList = ({ columns, classes, renderItemColumn, renderDetailsHeader, maxH
 
   return (
     <>
-      <Label className={classes.permissionLength}>
-        <FormattedMessage id='Permissions' />&nbsp;({permissions.length})
+      <Label className={classes.permissionLength} style={{marginLeft: '12px'}}>
+        <FormattedMessage id='Permissions' />
       </Label>
-      <Label className={classes.permissionText}>
+      <Label className={classes.permissionText} style={{marginLeft: '12px'}}>
         {!tokenPresent && <FormattedMessage id='sign in to consent to permissions' />}
-        {tokenPresent && <FormattedMessage id='permissions required to run the query' />}
+        {tokenPresent && <FormattedMessage id='permissions required to run the query'/>}
       </Label>
       <div
-        onMouseEnter={() => setIsHoverOverPermissionsList(true)}
-        onMouseLeave={() => setIsHoverOverPermissionsList(false)}>
+        onMouseEnter={() => {
+          // Check if the device width is less than 1260px
+          // OR if the browser width is less than 1290 ie. the browser is resized
+
+          if(screen.width < 1260 || window.innerWidth < 1290){
+            setIsScreenSizeReduced(true);
+          }
+        }
+        }
+        onMouseLeave={() => setIsScreenSizeReduced(false)}>
```

Comment:
Is this related to revoking consent?

Reply:
It is part of the ticket. Please refer to #2063